pkg/cmd/pipelineresource/list.go

```diff
@@ -17,12 +17,12 @@ package pipelineresource
 import (
 	"fmt"
 	"os"
-	"sort"
 	"strings"
 	"text/tabwriter"
 
 	"github.com/spf13/cobra"
 	"github.com/tektoncd/cli/pkg/cli"
+	pressort "github.com/tektoncd/cli/pkg/pipelineresource/sort"
```

Comment:
you can remove pressort here

Reply:
Why remove this ? It's being used to get the functions SortByNamespace and SortByTimeAndDate. :thinking: 
If you mean that sort should be used instead of pressort then I am just following convention from other resources where sorting libraries are named as trsort pr sort and such.

Reply:
alias is to differentiate when two packages are imported of same name, so just asked to remove the alias `presort`